test/integration/managementapi_cert_test.go

```diff
@@ -1,59 +0,0 @@
-package integration_test
-
-import (
-	"context"
-
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
-	"google.golang.org/protobuf/types/known/emptypb"
-
-	managementv1 "github.com/rancher/opni/pkg/apis/management/v1"
-	"github.com/rancher/opni/pkg/test"
-	_ "github.com/rancher/opni/plugins/example/test"
-)
-
-//#region Test Setup
-
-var _ = Describe("Management API Cerificate Management Tests", Ordered, Label("integration"), func() {
-	var environment *test.Environment
-	var client managementv1.ManagementClient
-	BeforeAll(func() {
-		environment = &test.Environment{}
-		Expect(environment.Start()).To(Succeed())
-		client = environment.NewManagementClient()
-	})
-
-	AfterAll(func() {
-		ExpectGracefulExamplePluginShutdown(environment)
-	})
-
-	//#endregion
-
-	//#region Happy Path Tests
-
-	It("can retrieve full certification chain information", func() {
```

Comment:
We should probably keep this test, since I remember it successfully catching (admittedly dumb) configuration errors when we made changes to the gateway cert logic

Reply:
Fixed + added the necessary fixes to the example plugin that was causing the original issues with this test